deployment/deploy/2000-deploy-LoanTokenModules.js

```diff
@@ -0,0 +1,49 @@
+const path = require("path");
+const { getContractNameFromScriptFileName } = require("../helpers/utils");
+const col = require("cli-color");
+const { getLoanTokenModulesNames } = require("../helpers/helpers");
+const func = async function (hre) {
+    const {
+        deployments: { deploy, get, log },
+        getNamedAccounts,
+        ethers,
+    } = hre;
+    const { deployer } = await getNamedAccounts(); //await ethers.getSigners(); //
+    let totalGas = ethers.BigNumber.from(0);
+
+    // @dev use to narrow down module contracts to redeploy
+    // e.g. you have three contracts modified but want to deploy only one
+    // then add the modules not ready for deployment to `dontDeployModules`
+    const dontDeployModules = {
+        /*
+        LoanTokenLogicLM: "LoanTokenLogicLM",
+        LoanTokenLogicWRBTC: "LoanTokenLogicWRBTC",
+        LoanTokenSettingsLowerAdmin: "LoanTokenSettingsLowerAdmin",
+        */
+    };
+    log(col.bgYellow("Deploying LoanTokenModules..."));
+    const modulesList = getLoanTokenModulesNames();
+    const loanTokenModuleNames = Object.keys(modulesList).filter(
+        (k) => !dontDeployModules.hasOwnProperty(k)
+    );
+    let modulesToAdd = {};
+    let modulesToReplace = {};
```

Comment:
unused vars 
in this case better to remove in order not to cause confusion

Reply:
Done